### PR TITLE
mysql-schema.md: update system tables

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8626
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tangenta — APPROVED** [review]

File: mysql-schema.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ These system tables contain grant information about user accounts and their priv
 - `tables_priv`: table-level privileges
 - `columns_priv`: column-level privileges
 - `password_history`: password change history
+- `default_roles`: the default roles for a user
+- `global_grants`: dynamic privileges
+- `global_priv`: the authentication information based on certificates
+- `role_edges` : the relationship between roles
```
Comment (shichun-0415) [inline mysql-schema.md:23]:
```suggestion
- `role_edges`: the relationship between roles
```

File: mysql-schema.md [hunk 2]
```diff
@@ -26,16 +30,29 @@ Currently, the `help_topic` is NULL.
...
 - `analyze_jobs`: the ongoing statistics collection tasks and the history task records within the last 7 days
 
+## Execution plan-related system tables
+
+- `bind_info`: the binding information of execution plans
+- `capture_plan_baselines_blacklist`: the blocklist for the automatic binding of the execution plan
+
 ## GC worker system tables
 
 - `gc_delete_range`: to record the data to be deleted
```
Comment (shichun-0415) [inline mysql-schema.md:50]:
```suggestion
- `gc_delete_range`: the KV range to be deleted
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by charlestonXdao — COMMENTED** [review]
good

**Pull request merged**